storage/database/leveldb_database.go

```diff
@@ -398,6 +409,30 @@ hasError:
 		// BlockCache size
 		db.blockCacheGauge.Update(int64(s.BlockCacheSize))
 
+		// Compaction related stats
+		compCount, err := db.db.GetProperty("leveldb.compcount")
+		if err != nil {
+			db.logger.Error("Failed to read database iostats", "err", err)
+			merr = err
+			continue
```

Comment:
Doesn' it causes continuous execution of `for` loop without refresh time?

Reply:
@aidan-kwon You're right. Other parts of the function have been changed a lot. Let me check the other part, too.

Reply:
Please compare the current geth code.
I think reference PR is old. After that there is some changes like https://github.com/ethereum/go-ethereum/pull/16794.

Reply:
`meter` function of Klaytn is different from the one of Geth's, since Geth still uses `GetProperty` function to retrieve the stats, while Klaytn uses `Stats` function, which provides LevelDB statistics by `leveldb.DBStats` object.